test/greet_test.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var expect = require('chai').expect;
+var greet = require("../greet.js");
+
+describe('greet.js', function() {
+
+  it('should greet someone', function() {
+    expect(greet.greet('Stephanie')).to.eql('hello Stephanie');
+  });
+});
```

Comment:
Using "greet.greet" can be a little confusing. Perhaps change the "greet" variable name defined previously.